ui/src/app/components/help-sidebar.tsx

```diff
@@ -185,6 +185,7 @@ interface State {
   tooltipId: number;
   currentModal: CurrentModal;
   runtimeErrors: Array<RuntimeError>;
+  runtimeConfPanelInitialState: PanelContent | null;
```

Comment:
Why initialState and not just state?

Reply:
Runtime Conf has a lot of internal logic about what its Panel state should be.  This only sets what we want its initial state to be.